cpp/include/cudf_test/column_utilities.hpp

```diff
@@ -198,15 +199,14 @@ std::pair<thrust::host_vector<T>, std::vector<bitmask_type>> to_host(column_view
 template <>
 inline std::pair<thrust::host_vector<std::string>, std::vector<bitmask_type>> to_host(column_view c)
 {
-  auto strings_data = cudf::strings::create_offsets(strings_column_view(c));
-  thrust::host_vector<char> h_chars(strings_data.first.size());
-  thrust::host_vector<size_type> h_offsets(strings_data.second.size());
-  CUDA_TRY(
-    cudaMemcpy(h_chars.data(), strings_data.first.data(), h_chars.size(), cudaMemcpyDeviceToHost));
-  CUDA_TRY(cudaMemcpy(h_offsets.data(),
-                      strings_data.second.data(),
-                      h_offsets.size() * sizeof(cudf::size_type),
-                      cudaMemcpyDeviceToHost));
+  auto const scv     = strings_column_view(c);
+  auto const h_chars = cudf::detail::make_std_vector_sync<char>(
+    cudf::device_span<char const>(scv.chars().data<char>(), scv.chars().size()),
+    rmm::cuda_stream_default);
+  auto const h_offsets = cudf::detail::make_std_vector_sync(
+    cudf::device_span<cudf::offset_type const>(
+      scv.offsets().data<cudf::offset_type>() + scv.offset(), scv.size() + 1),
+    rmm::cuda_stream_default);
```

Comment:
Humm, I think we should better use 2 calls to `make_vector_async` then call stream sync, because 2 calls to `make_vector_sync` will make 2 implicit stream syncs.

Nevertheless, this is in the test namespace thus I'm not worried much about performance.